src/main/java/org/jabref/logic/importer/AuthorListParser.java

```diff
@@ -96,6 +97,9 @@ private static StringBuilder buildWithAffix(Collection<Integer> indexArray, List
     public AuthorList parse(String listOfNames) {
         Objects.requireNonNull(listOfNames);
 
+        // Handle the statement "and others" at the end, removing it
+        listOfNames = StringUtil.removeStringAtTheEnd(listOfNames.trim(), " and others");
```

Comment:
We should introduce a special author `OTHERS` as `static final` instance of [`Author`](https://github.com/JabRef/jabref/blob/2f84f1cb71503c7fa6c7f9fa3b8ad8e12beee914/src/main/java/org/jabref/model/entry/Author.java#L14) and should be returned as last element in the list -- if that string is encountered.

This is also important as pre-condition for https://github.com/JabRef/jabref/issues/9439

Reply:
I'm not sure if I understood: do you mean that, for instance, the following call should return the following result (in pseudo-code, just to understand)?
```
AuthorListParser.parse("Alexander Artemenko and others") -> AuthorsList[
new Author("Alexander", "A.", null, "Artemenko", null), 
new Author("others", "oth", null, null, null)
]
```
Hence, two Authors with the last of them a "fictious" author named "others"? Is it right?

Reply:
```java
AuthorListParser.parse("Alexander Artemenko and others") -> AuthorList.of(
new Author("Alexander", "A.", null, "Artemenko", null), 
Author.OTHERS
)
```

And in Author:

```
public static final OTHERS = new Author("others", "", null, null, null)
```

Note that I did not put `oth` there, because it is `others` in BibTeX, not `others, oth` or other strings not equal to `others`.

## Background

In my reply, I thought: With `Author.OTHERS` defined in the class `Author` as `public static final OTHERS = new Author(...)`.

Having read [Effective Java](https://github.com/HugoMatilla/Effective-JAVA-Summary#3-enforce-the-singleton-property-with-a-private-constructor-or-an-enum-type) Item 3, `enum` cannot help here.